department-of-veterans-affairs/vets-website
# VAGOV-3223: Homepage and all of the pages that live inside Locations & Services and News & Events

## Description
updates "facility" section of va.gov to follow responsive designs

## Testing done
locally with staging data.

1. build site
2. have access to this zeplin project https://app.zeplin.io/project/5d014fd159c9931608872c98/dashboard
3. browse around the pittsburgh-health-care site
4. check responsiveness (> 768, 767 - 481, and < 481)
### Region Home
http://localhost:3001/pittsburgh-health-care/
### Locations List
http://localhost:3001/pittsburgh-health-care/locations/
### Location Detail
http://localhost:3001/pittsburgh-health-care/locations/hj-heinz/
### Health Services
http://localhost:3001/pittsburgh-health-care/health-services/
### Events List
http://localhost:3001/pittsburgh-health-care/events/
### Event Detail
http://localhost:3001/pittsburgh-health-care/events/women-veterans-tea/
### Stories List
http://localhost:3001/pittsburgh-health-care/stories/
### Story Detail
http://localhost:3001/pittsburgh-health-care/stories/grand-opening-held-relocated-belmont-clinic/
### News Releases List
http://localhost:3001/pittsburgh-health-care/press-releases/
### News Release Detail
http://localhost:3001/pittsburgh-health-care/press-releases/washington-county-va-outpatient-clinic-move-larger-space/
### Leadership
http://localhost:3001/pittsburgh-health-care/leadership/
### Leadership detail
http://localhost:3001/va-central-office/barbara-forsha/

## Acceptance criteria
https://va-gov.atlassian.net/browse/VAGOV-3223

## Definition of done
